tests/smoketest/smoketest_job.yaml.template

```diff
@@ -43,7 +45,7 @@ spec:
           allowPrivilegeEscalation: false
 
       - name: smoketest-ceilometer
-        image: quay.io/tripleomaster/openstack-ceilometer-notification:current-tripleo
+        image: image-registry.openshift-image-registry.svc:5000/<<NAMESPACE>>/openstack-ceilometer-notification:latest
```

Comment:
I wonder if doing this against `latest` is necessary any more? Or does `latest` == `wallaby` at this point effectively?

I'd had for this to start breaking if some of these images end up getting used in next-gen. Is there a wallaby tag we should follow instead?

Reply:
Ahhh! Transposed the wrong thing in my head :)

current-tripleo sounds like it should be latest of whatever we're shipping I guess? 👍🏻